Makefile

```diff
@@ -119,9 +119,34 @@ scratch-rpm: $(RPM_SPECFILE) archive
 build-in-podman: archive
 	podman build --security-opt seccomp=unconfined -t weldr-client:$(VERSION) -f Containerfile.test .
 
+DOCKER_IMAGE_CLI ?= osbuild-cli_dev
+DOCKERFILE_CLI ?= Containerfile.dev
+
+# either "docker" or "sudo podman"
+# podman needs to build as root as it also needs to run as root afterwards
+CONTAINER_EXECUTABLE ?= docker
```

Comment:
This should be podman, not docker. Same for the `DOCKER` variable names :)

Reply:
the reason for both is:
* starting from "[getting started `make run_service_no_fronted`](https://github.com/schuellerf/osbuild-getting-started/blob/HMS-3883-fully-source-containers/Makefile#L148)" I couldn't get it working with podman (and was `docker` in this repo even before I started reviving it)
* also this whole stack needs `sudo podman` (if it would work with `podman`) and thus needs it on all layers due to the user-centric behavior of `podman`
but as I'm cleaning stuff up and can override from the top anyway - I'll fix it here and remove the comment as for this layer it's not really needed (standalone)

Reply:
What fails if you don't use sudo with podman? I was at least able to do a `make container.dev` with those small changes and run the container (but didn't do more than confirm it ran and has composer-cli in it)